api/proto/teleport/integration/v1/awsoidc_service.proto

```diff
@@ -276,6 +281,36 @@ message ListSubnetsResponse {
   string next_token = 2;
 }
 
+// ListVPCsRequest is a request for a paginated list of AWS VPCs.
+message ListVPCsRequest {
+  // Integration is the AWS OIDC Integration name.
+  // Required.
+  string integration = 1;
+  // Region is the AWS Region
+  // Required.
+  string region = 2;
+  // NextToken is the token to be used to fetch the next page.
+  // If empty, the first page is fetched.
+  string next_token = 3;
```

Comment:
Do we want to include page size ?

Reply:
Default quota for VPCs is 5.
However, it can be increased beyond the page size